tests/ok_comm/test_ok_comm_barcode.py

```diff
@@ -25,10 +25,13 @@
     "communication_type": "barcode_comm",
     "command": "start_scan",
 }
+
+# Tanner (12/30/20): All barcodes must be twelve characters long
```

Comment:
minor note: please add some explanation about *why* all barcodes must be 12 (i.e. this is a requirement of the hardware/firmware, not something fundamental within the software)